src/main/java/com/puppycrawl/tools/checkstyle/checks/NewlineAtEndOfFileCheck.java

```diff
@@ -88,6 +88,19 @@
  * <pre>
  * &lt;module name=&quot;NewlineAtEndOfFile&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * // File ending with a new line
+ * public class Test {⤶
+ * ⤶
+ * }⤶
+ * ⤶// ok
```

Comment:
This file ends with two `⤶` symbols. It should be simple
```
 }⤶// ok
```
For clarity, please add a note like
The `// ok` comment is "virtual", not actually present in the file.

Reply:
Done. Please review